include/jemalloc/internal/extent_inlines.h

```diff
@@ -211,6 +220,12 @@ extent_szind_set(extent_t *extent, szind_t szind) {
 }
 
 JEMALLOC_INLINE void
+extent_nfree_set(extent_t *extent, unsigned nfree) {
```

Comment:
The nfree field is invalid unless the extent is a slab, so the inc/dec will work okay, but I don't want to initialize nfree in ```extent_init()```, nor create a special ```extent_slab_init()```.